skbeam/core/tests/test_correlation.py

```diff
@@ -102,7 +115,20 @@ def test_lazy_multi_tau():
     for second_half_result in gen_second_half:
         pass
 
-    assert np.all(full_result.g2 == second_half_result.g2)
+    assert np.all(full_result.g2 ==
+                  second_half_result.g2)
+
+
+def test_two_time_corr():
+    setup()
+    y = []
+    for i in range(50):
+        y.append(img_stack[0])
+    two_time, lag_steps = two_time_corr(rois,
+                                        np.asarray(y), 50,
+                                        num_bufs=50, num_levels=1)
+    assert np.all(two_time)
```

Comment:
I would like this test to be more like the lazy_one_time test, where the first half of the images are correlated, then the second half are correlated with the internal state from the first half. Then the results of the second half correlation can be compared with the results of running the correlation with the full stack.  That is one effective way of testing to make sure we are capturing the correct amount of state from the lazy_two_time function